script.js

```diff
@@ -1,4 +1,79 @@
+  if (userChoice == scissorsPaperStoneResponse) {
+    return `${userName}, you chose ${userChoice} and the computer chose ${scissorsPaperStoneResponse} - its a draw! You have won ${numberPlayerWins} times and the computer has won ${numberComputerWins} times.`;
+  }
+  if (userChoice == "scissors" && scissorsPaperStoneResponse == "paper") {
+    numberPlayerWins = numberPlayerWins + 1;
+    return `${userName}, you chose ${userChoice} and the computer chose ${scissorsPaperStoneResponse} - you win! You have won ${numberPlayerWins} times and the computer has won ${numberComputerWins} times.`;
+  }
+  if (userChoice == "scissors" && scissorsPaperStoneResponse == "stone") {
+    numberComputerWins = numberComputerWins + 1;
+    return `${userName}, you chose ${userChoice} and the computer chose ${scissorsPaperStoneResponse} - you lose! You have won ${numberPlayerWins} times and the computer has won ${numberComputerWins} times.`;
+  }
+  if (userChoice == "paper" && scissorsPaperStoneResponse == "stone") {
+    numberPlayerWins = numberPlayerWins + 1;
+    return `${userName}, you chose ${userChoice} and the computer chose ${scissorsPaperStoneResponse} - you win!You have won ${numberPlayerWins} times and the computer has won ${numberComputerWins} times.`;
+  }
+  if (userChoice == "paper" && scissorsPaperStoneResponse == "scissors") {
+    numberComputerWins = numberComputerWins + 1;
+    return `${userName}, you chose ${userChoice} and the computer chose ${scissorsPaperStoneResponse} - you lose! You have won ${numberPlayerWins} times and the computer has won ${numberComputerWins} times.`;
+  }
+  if (userChoice == "stone" && scissorsPaperStoneResponse == "scissors") {
+    numberPlayerWins = numberPlayerWins + 1;
+    return `${userName}, you chose ${userChoice} and the computer chose ${scissorsPaperStoneResponse} - you win! You have won ${numberPlayerWins} times and the computer has won ${numberComputerWins} times.`;
+  }
+  if (userChoice == "stone" && scissorsPaperStoneResponse == "paper") {
+    numberComputerWins = numberComputerWins + 1;
+    return `${userName}, you chose ${userChoice} and the computer chose ${scissorsPaperStoneResponse} - you lose! You have won ${numberPlayerWins} times and the computer has won ${numberComputerWins} times.`;
+  }
+  if (
+    userChoice != "scissors" &&
+    userChoice != "paper" &&
+    userChoice != "stone"
+  ) {
+    return "Don't you know how to play scissors paper stone??";
+  }
+};
+
+// run main function
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  var myOutputValue = "";
+  if (currentGameMode == "waiting for player name") {
+    userName = input;
+    currentGameMode = "Scissors Paper Stone";
+    myOutputValue = `Hello ${userName}`;
+    return myOutputValue;
+  } else if (currentGameMode == "Scissors Paper Stone") {
+    myOutputValue = scissorsPaperStone(userName, input);
+    return myOutputValue;
+  }
```

Comment:
great job using global variable to get player name and if statement to change game mode. good usage of function scissorPaperStone to generate outputValue